src/main/java/org/commcare/formplayer/Application.java

```diff
@@ -44,25 +44,6 @@ public static void main(String[] args) {
         ReferenceHandler.setUseThreadLocalStrategy(true);
     }
 
-    /**
-     * This filter intercepts responses before they're dispatched and logs the request URL and response status
-     */
-    private class ResponseLoggingFilter extends GenericFilterBean {
-        @Override
-        public void doFilter(ServletRequest request, ServletResponse response, FilterChain filterChain) throws IOException, ServletException {
-            final HttpServletRequest httpRequest = (HttpServletRequest) request;
-            final HttpServletResponse httpResponse = (HttpServletResponse) response;
-            log.info("Got request URL: " + httpRequest.getRequestURL() + " , response code: " + httpResponse.getStatus());
-            filterChain.doFilter(request, response);
-        }
-    }
-
-    // Autowire the filter above
-    @Bean
-    public Filter loggingFilter() {
```

Comment:
is this not used anywhere?

Reply:
We have 2 logging filters and one logging aspect:

1. ResponseLoggingFilter (which is the one removed here)
  - A very generic logging filter that just outputs the url and 200 (the status code is always 200 since the logging happens before the request is processed)
2. RequestResponseLoggingFilter (enabled if `sensitiveData.enableLogging` is true)
  - logs the full request and response
3. LoggingAspect (runs around every view method call)
  - logs basic info about the request

I don't know why we have all these different loggers. (1) is clearly of very little use which is why I'm removing it.